fe/fe-core/src/main/java/com/starrocks/connector/Connector.java

```diff
@@ -44,4 +44,12 @@ default void bindConfig(ConnectorConfig config) {
     default boolean supportMemoryTrack() {
         return false;
     }
+
+    default void init() {
```

Comment:
I think we don't need to add this as virtual function of base class `Connector`. Only `LazyConnector` should have a `init` function

Reply:
Why not? looks like the delegator also need `init` function? Looks like it's reasonable to have this `init` function.